pytest.ini

```diff
@@ -72,6 +72,10 @@ qt_log_ignore =
     ^[^ ]*qtwebengine_dictionaries'$
     # Qt 5 on Archlinux
     ^QSslSocket: cannot resolve .*
+    # Seems to happen after we try to complete immediately after clearing a
+    # model (eg when no completion function is available for the current text
+    # pattern.
```

Comment:
```suggestion
    # model (e.g. when no completion function is available for the current text
    # pattern).
```

Reply:
I changed it to "for example", with no parenthesis. Because I try to avoid both loan words and acronyms when I'm feeling patient. Not sure what I was thinking using both in one go. Hope that's okay!